packages/cloud_firestore/cloud_firestore/example/ios/Runner/GoogleService-Info.plist

```diff
@@ -35,4 +35,4 @@
 	<key>DATABASE_URL</key>
 	<string>https://flutterfire-e2e-tests-default-rtdb.europe-west1.firebasedatabase.app</string>
 </dict>
-</plist>
\ No newline at end of file
+</plist>
```

Comment:
can also remove this change.

Reply:
Honestly, I am not sure what changed here. I literally tried copying the code from master and pasting it. I'm confused

Reply:
Done :D